ddpui/ddpprefect/prefect_service.py

```
@@ -751,3 +759,187 @@
     return res
+
+
+def get_late_flow_runs(payload: FilterLateFlowRunsRequest) -> list[dict]:
+    res = prefect_post(f"flow_runs/late", payload.dict())
+
+    return res["flow_runs"]
+
+
+def post_prefect_workers_count(work_queue_name: str, work_pool_name: str) -> list[dict]:
+    res = prefect_post(
+        f"workers/filter/",
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)